main.css

```diff
@@ -0,0 +1,262 @@
+html {
+  background-color: #333;
+  color: white;
+  font-family: "Balsamiq Sans", cursive;
+  margin: 0;
+  width: 100%;
+}
+
+.nav-bar1 {
+  align-items: center;
+  background-color: rgb(136, 136, 136);
+  height: 100px;
+  text-align: center;
+}
+
+.nav-list > li {
+  display: inline;
+  font-size: 20px;
+  list-style: none;
+}
+
+.nav-item {
+  color: white;
+  margin: 25px;
+  text-decoration: none;
+}
+
+.logo {
+  border-radius: 5px;
+  width: 100px;
+}
+
+.container-grid {
+  background-color: rgb(45, 45, 46);
+  display: grid;
+  grid-gap: 2px;
+  grid-template-columns: auto auto;
+  grid-template-rows: 600px 600px;
+  margin: 0 auto;
+  text-align: center;
+  width: 100%;
+}
+
+.grid1,
+.grid2 {
+  background-color: rgb(45, 45, 46);
+}
+
+.grid3,
+.grid4 {
+  background-color: rgb(66, 66, 66);
+}
```

Comment:
This selector has the same style definition of your selector on line 89, Group selectors in order to minimize code in your CSS file.